### PR TITLE
Update urfave/cli to v2.0.0

### PR DESCRIPTION
**Pull request by saschagrunert**
The v2 introduces changes which need code adaptions on the CRI-O side. This PR fixes them as well.

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Comment by saschagrunert** [comment]
Rebased on top of the master to see if this fixes the CI.

**Comment by giuseppe** [comment]
LGTM

File: docs/crio.8.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 # GLOBAL OPTIONS
 
-**--additional-devices**="": Devices to add to the containers (default: [])
+**--additional-devices**="": Devices to add to the containers (default: []) (default: [])
```
Comment (TomSweeneyRedHat) [inline docs/crio.8.md]: I'm not sure what the double default means here and at line 120?

Comment (TomSweeneyRedHat) [reply docs/crio.8.md]: make that through out.

Comment (saschagrunert) [reply docs/crio.8.md]: Ah I see, let me check if I can fix this...

**Pull request merged**